app/views/layouts/_protect.html.haml

```diff
@@ -22,4 +22,5 @@
   - if @politems
     - @embedded = true
     - @quadicon_no_url = true
-    = render :partial => "layouts/gtl"
+    - no_flash_div = true
+    = render :partial => "layouts/gtl", :locals => {:no_flash_div => no_flash_div}
```

Comment:
Would the simpler `= render :partial => "layouts/gtl", :locals => {:no_flash_div => true}` work?